src/components/About/icons/Values/Group/index.tsx

```diff
@@ -0,0 +1,43 @@
+const GroupIcon = ({ width = 96, height = 76, className = "" }) => {
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox="0 0 96 76"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={`${className}`}
+    >
```

Comment:
First off, loving the SVG as React components. I do think though we could standardize some of the attributes here a bit better. Here's a suggestion for accessibility.

```suggestion
    <svg
      aria-hidden="true"
      aria-labelledby="new-tab-icon-title new-tab-icon-description"
      role="img"
      width={width}
      height={height}
      viewBox="0 0 96 76"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      className={`${className}`}
    >
```

- https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/
- https://css-tricks.com/accessible-svg-icons/

Reply:
Looks like committing aria-hidden=true and aria-labelledby would be contradictory. I think it's ok if screen readers skip the icons since it's just an embellishment and the sentiment is covered by the title and description labels in the cards.